### PR TITLE
Add, remove and edit implementation 

### PR DESCRIPTION
**Pull request by Otegaa**
In this PR, I implemented a function for adding, removing, and editing list items on the page. 
I sent this review some hours ago and the previous reviewer uploaded changes required without writing any changes, which cost me a review so I am currently on my last review.

### CODE REVIEW

File: modules/addRemove.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    getStorage.forEach((item) => {
+      list.innerHTML += `
+         <li class="list-task">
+            <div class="to-do-list-task">
+              <input type="checkbox" class="to-do-input" id= ${item.index}/>
+              <p class="to-do-text" id= ${item.index}>${item.desc}</p>
+            </div>
+            <div class="to-do-btn-del">
+              <i class="fas fa-ellipsis-v fa-2x remove-list" id= ${item.index}  ></i>
+              <i class='fa-solid fa-trash-can fa-2x hidden add-trash' id= ${item.index}></i>
```
Comment (Felix45) [inline modules/addRemove.js]: - [ Required ] Great job implementing the `add` and `edit` functionalities, however, when I tried to click the `delete` button I was not able to delete the to-do list item I had clicked on. Please see the video below and rectify this. Also, remember that when deleting a task it should update all remaining items' `indexes`, so they represent the current list order and are unique.

## Video
https://user-images.githubusercontent.com/16392046/229125250-612d298f-5111-4db7-94dc-fa7000f90c43.mp4

**Review by Felix45 — CHANGES_REQUESTED** [review]
Hi @Otegaa , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :heavy_check_mark: 
- [x] No linter errors :white_check_mark: 
- [x] Good UI design :white_check_mark: 
- [x] Good descriptive commit messages :white_check_mark:  
- [x] GitHub flow is used correctly :white_check_mark:   
- [x] Repository has a professional readme file :white_check_mark:  

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/addRemove.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+  list.addEventListener('click', (e) => {
+    const target = e.target.closest('.add-trash');
+    if (!target) return;
+    removeListIndex(target.id);
+    getStorage.forEach((item, i) => {
+      item.index = i + 1;
+    });
+    setStorage();
+  });
```
Comment (jerryowusu) [inline modules/addRemove.js:63]: - [ ] Nice job so far. However, as indicated by the previous reviewer, the delete functionality is not working as it is suppose to. When you click on the three dots to select the delete icon, it automatically activates the edit functionality hence making it difficult to click on the delete button and delete a task. Kindly check and make sure the delete button and functionality works to remove a task successfully. 

You can achieve this by setting the edit event listener on the text of the task to edit/update and not the whole div (which includes the delete button icon)

- [ ]  Also, remember that when deleting a task it should update all remaining items' indexes, so they represent the current list order and are unique.

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Otegaa 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

noooo


 ## Highlights
Add function working fine :heavy_check_mark: 
Great effort so far :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

- [ ] For the social media handles shown below, kindly consider using your actual social media  names to make it easy to identify and remember instead of @githubhandle, @twitterhandle etc.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/addRemove.js (hunk 2 above)
Comment (Otegaa) [reply modules/addRemove.js:63]: I think you should check the video and then my project again. When you click on the ellipsis, it activates edit and delete. The user has a choice to do either of them and it is working perfectly well. Pls, check from your end.

**Review by OLIPLICHE — APPROVED** [review]
Hi @Otegaa, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights
- Implemented the changes requested by the previous review 👍 
- Linter is green (the last commit has a green check mark (✅)) 👏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**